outputs.tf

```
@@ -0,0 +1,3 @@
+output "pipeline_arn" {
```

```suggestion
output "cloudformation_stack_pipeline_arn" {
```

Per eng standards this should be changed.. but feel free to reject this suggestion. I think this might be an edge case that we should instead update the eng standards for. Putting the resource type doesn't make much sense when the resource is a container for other resources.

---

I'd argue two things:
1. We're trying to obfuscate the fact that this is using CFN as much as possible
2. The output isn't a `cloudformation_stack_pipeline_arn`. The CFN stack has its own ARN, but we're reaching deeper into the CFN stack output to get the actual pipeline ARN

---

I agree. https://github.com/rhythmictech/engineering-standards/issues/5